app/components/Search/SearchPageResults.css

```diff
@@ -37,6 +37,8 @@
 
 .searchResultItemIcon {
   margin-left: 10px;
+  font-size: 0.3rem;
```

Comment:
The icons comp has a `size` prop u know

Reply:
This is for the Alt text. Not the image. It needs to be really small to not look crazy as hell